chaco/plots/multi_line_plot.py

```diff
@@ -484,7 +484,7 @@ def _render(self, gc, line_points, selected_points=None):
                 # Existence of self.color_func overrides self.color.
                 color_func = self.color_func
             else:
-                color_func = lambda k: self.color_
+                def color_func(k): return self.color_
```

Comment:
This looks like an unrelated change, but put a line break after the `:` for the function body.